File: PasswordLibrary/Config/Service.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PasswordLibrary.Config
{
    public class Service
    {
        public String Name { get; set; }
        public int Size { get; set; }
        public bool IncludeEspecialChar { get; set; }
        public bool IncludeNumbers { get; set; }
        public bool IncludeUpperLower { get; set; }
        public bool OnlyNumbers { get; set; }
    }
}

File: PasswordWeb/ClientApp/src/data/Service.ts
    export class Service
    {
       constructor() {
            this.Name = "";
            this.Size = 15;
            this.IncludeEspecialChar = true;
            this.IncludeNumbers = true;
            this.IncludeUpperLower = true;
            this.OnlyNumbers = false;
        }

        Name: String;
        Size: number;
        IncludeEspecialChar: boolean; 
        IncludeNumbers: boolean; 
        IncludeUpperLower: boolean; 
        OnlyNumbers: boolean; 
    }